modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportAnimalQTL.pm

```diff
@@ -0,0 +1,345 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+
+=head1 ImportAnimalQTL
+
+This module import AnimalQTL data. The module relies on the existance of the files to be imported.
+
+# Currently human download is needed as a licence aggreement is needed
+# steps:
+# 1. follow link https://www.animalgenome.org/cgi-bin/QTLdb/index
+# 2. select species e.g. Chicken QTL
+# 3. Downloads -> 'All data by bp' gff format
+# 4. get the *.txt.gz file and rename to species(_assembly)?.gff3.gz eg. gallus_gallus_GRCg6a.gff3.gz or gallus_gallus.gff3
+
+# NOTE: make sure the Animal QTL db has the data mapped to the Ensembl assembly name
+
+=cut
+
+
+package Bio::EnsEMBL::Variation::Pipeline::PhenotypeAnnotation::ImportAnimalQTL;
+
+use strict;
+use warnings;
+
+use File::Path qw(make_path);
+use File::stat;
+use POSIX 'strftime';
+
+use base ('Bio::EnsEMBL::Variation::Pipeline::PhenotypeAnnotation::BasePhenotypeAnnotation');
+
+my %source_info;
+
+# AnimalQTLdb URL set up for Ensembl: https://www.animalgenome.org/QTLdb/export/ENS83H19HZS/
+#my $animalqtl_url = 'https://www.animalgenome.org/cgi-bin/QTLdb/index';
+my %animalQTL_species_url = (
+  gallus_gallus => 'https://www.animalgenome.org/QTLdb/export/ENS83H19HZS/QTL_GG_5.0.gff.txt.gz', #Gallus gallus
```

Comment:
Could use a variable to store the root of the AnimalQTL URL: `https://www.animalgenome.org/QTLdb/`
Actually, could it be stored into `Constants.pm` ?